### ISSUE TITLE
first_service_* fields in fxa_users_services_first_seen_v1 should reflect the first value in the associated flow_id

### ISSUE THREAD

**Issue by irrationalagent** [mozilla/bigquery-etl#769]
In `fxa_users_services_first_seen_v1` the values for the fields listed below currently reflect the value at the point of flow completion, rather than the value at the beginning of the flow. 

The table is meant to reflect the time, OS, etc of the device during a user's first usage or authentication with a service (which may also be their registration flow if they are signing up for a new account; this is reflected in the `did_register` variable).

The tricky thing here is that over the course of an authentication flow the value (for example) of `OS` can actually change. For example, a user can be registering for a new account on desktop, but confirm their email address (required in order to finish account set-up) by clicking the confirmation email on their phone. So the initial events in the registration flow might be associated with `Windows` for OS but the final events (e.g. the registration complete event) might be associated with `iOS`. 

In the above example, we should really attribute the registration to `Windows` not `iOS`. But with the way [the ETL is coded now](https://github.com/mozilla/bigquery-etl/blob/master/sql/telemetry_derived/fxa_users_services_first_seen_v1/init.sql#L34), we are taking the value at the time of completion, which means the above example will be attributed to `iOS`. The affected variables are:

first_service_country
first_service_flow
first_service_os
first_service_timestamp 

Importantly, `entrypoint` actually behaves as desired. This is because the script joins back on the original table to take the first value of `entrypoint` associated with the flow's `flow_id`.

TLDR - for `fxa_users_services_first_seen_v1` we should aggregate the variables listed above in the way `entrypoint` is aggregated. If we fix this it will require a backfill. The impact of this bug is that we will over-attribute too many registrations and first-service-usages to mobile (especially historical data from the period FxA was still using confirmation links rather than codes).

**Comment by irrationalagent** [comment]
@ksiegler1 this may be a bug of the type we talked about yesterday. no need to jump on it ASAP but could be worth a look.

**Comment by jklukas** [comment]
These tables have been deprecated.
